src/main/resources/staff/templates/document/countyCourtJudgmentDetails.html

```diff
@@ -7,9 +7,33 @@
       size: 8.5in 11in;
       margin: 10%;
 
+      @bottom-center {
+        content: "For further details of the courts www.gov.uk/find-court-tribunal";
+        font-size: 10px;
+      }
+
+      @bottom-right {
+        content: "© Crown copyright Page " counter(page) " of " counter(pages);
+        /*noinspection CssNoGenericFontName*/
```

Comment:
please remove commented out code.

Reply:
that's not commented out code, that's for intelliJ to not complain

Reply:
ok